template/partials/report.js

```diff
@@ -1,61 +1,87 @@
 const prettyBytes = require('pretty-bytes')
+var moment = require('moment')
+function datestuff (date) {
+  return moment(date).format('MMMM Do YYYY, h:mm:ss a')
+}
+
+function diff (finish, start) {
+  var finishTime = new Date(finish)
+  var startTime = new Date(start)
+  var difference = finishTime.getTime() - startTime.getTime()
+  difference = Math.floor(difference / 1000)
+  return difference
+}
 
 module.exports = function (results, hx) {
   return hx`
-  <div class='report'>
-    ${results.title ? hx`<h1>Results for ${results.title}</h1>` : hx`<h1>Autocannon results</h1>`}
-    <div class='latency'>
-      <h2>Latency</h2>
-      <table style="width:100%">
-        <tr>
-          <th>Stat</th>
-          <th>Value</th>
-        </tr>
-        ${
-          Object.keys(results.latency).map((key) => {
-            return hx`<tr>
-              <td>${key}</td>
-              <td>${results.latency[key]}</td>
-            </tr>`
-          })
-        }
-      </table>
+  <div>
+    <div class='title'>
+      ${results.title ? hx`<h1>Results for ${results.title}</h1>` : hx`<h1>Autocannon results</h1>`}
     </div>
-    <div class='throughput'>
-      <h2>Throughput</h2>
-      <table style="width:100%">
-        <tr>
-          <th>Stat</th>
-          <th>Value</th>
-        </tr>
-        ${
-          Object.keys(results.throughput).map((key) => {
-            return hx`<tr>
-              <td>${key}</td>
-              <td>${prettyBytes(results.throughput[key])}</td>
-            </tr>`
-          })
-        }
-      </table>
-      <div class='meta'>
-        <h2>General info</h2>
-        <ul>
-          <li><b>Errors:</b> ${results.errors}</li>
-          <li><b>Timeouts:</b> ${results.timeouts}</li>
-          <li><b>Runtime:</b> ${results.duration}</li>
-          <li><b>Start time:</b> ${new Date(results.start)}</li>
-          <li><b>Finish time:</b> ${new Date(results.finish)}</li>
+    <div class='report'>
+      <div class ='object content no-border'>
+        <ul class ='grid'>
+          <li><b>Start Time:</b> ${datestuff(results.start)} </li>
           <li><b>Connections:</b> ${results.connections}</li>
+          <li><b>Timeouts:</b> ${results.timeouts}</li>
+          <li><b>Throughput average:</b> ${prettyBytes(results.throughput.average)}/sec</li>
+        </ul>
+        <ul class ='grid'>
+          <li><b>Finish Time:</b> ${datestuff(results.finish)}</li>
           <li><b>Pipelining:</b> ${results.pipelining}</li>
-          <li><b>1xx responses:</b> ${results['1xx']}</li>
-          <li><b>2xx responses:</b> ${results['2xx']}</li>
-          <li><b>3xx responses:</b> ${results['3xx']}</li>
-          <li><b>4xx responses:</b> ${results['4xx']}</li>
-          <li><b>5xx responses:</b> ${results['5xx']}</li>
+          <li><b>Requests Average:</b> ${results.requests.average} reqs/sec</li>
         </ul>
+        <ul class='grid'>
+          <li><b>Duration:</b> ${diff(results.finish, results.start) + ' sec(s)'}</li>
```

Comment:
Diff isn't needed, checkout result.duration. We do the same calculation for you in autocannon.